store/tikv/coprocessor.go

```diff
@@ -1312,37 +1313,31 @@ type rateLimitAction struct {
 	// totalTokenNum indicates the total token at initial
 	totalTokenNum uint
 	cond          struct {
-		*sync.Cond
+		sync.Mutex
 		// exceeded indicates whether have encountered OOM situation.
 		exceeded bool
 		// remainingTokenNum indicates the count of tokens which still exists
 		remainingTokenNum uint
 		// isTokenDestroyed indicates whether there is one token has been isTokenDestroyed after Action been triggered
 		isTokenDestroyed bool
 		once             sync.Once
-		// triggered indicates whether the action is triggered
-		triggered bool
-		// waitingWorkerCnt indicates the total count of workers which is under condition.Waiting
-		waitingWorkerCnt uint
 		// triggerCountForTest indicates the total count of the rateLimitAction's Action being executed
 		triggerCountForTest uint
 	}
 }
 
-func newRateLimitAction(totalTokenNumber uint, cond *sync.Cond) *rateLimitAction {
+func newRateLimitAction(totalTokenNumber uint) *rateLimitAction {
 	return &rateLimitAction{
 		totalTokenNum: totalTokenNumber,
 		cond: struct {
-			*sync.Cond
+			sync.Mutex
 			exceeded            bool
 			remainingTokenNum   uint
 			isTokenDestroyed    bool
```

Comment:
We can remove isTokenDestroyed and only use exceeded.

Reply:
updated.